Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:20-alpine3.18 as base
+FROM node:20-alpine3.16 as base
```

Comment:
Is this downgrade deliberate?

Reply:
Yes - there is a bug introduced in 3.17 that causes errors in docker-compose. 3.17/3.18 are seemingly less stable and are having frequent updates, so downgraded at least for now.